packages/suite/src/components/guide/GuideButton.tsx

```diff
@@ -23,18 +23,23 @@ const Wrapper = styled.button`
     box-shadow: 0 2px 7px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_15},
         0 2px 3px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_5};
     transition: opacity 0.3s ease 0.3s;
+    opacity: ${({ $isGuideOpen }) => ($isGuideOpen ? 0 : 1)};
+
+    &:focus {
+        transition-delay: 0s; /* hide button without delay on guide open */
+    }
 
     & > img {
         display: block;
     }
 `;
 
 export const GuideButton = () => {
```

Comment:
Not a fan of how the button stays on top of the guide when it opens :c Is it possible to either make it disappear faster or be underneath the guide?

Reply:
Ok, I disabled the opacity transition when clicking the button. 167ab5c587e39af124eff83339a824da5b5efbb1

Reply:
Please, add this on focus, so that the button disappears a bit softer :) It should not overlap with it. Otherwise, all good! :)

```
transition: opacity 0.1s ease;
```

Reply:
Done! I rebased already.